### ISSUE TITLE
Clarify "8.1. The copy action" in regards to copying selected data

### ISSUE THREAD

**Issue by danburzo** [w3c/clipboard-apis#49]
I've initially logged it here https://github.com/w3c/editing/issues/168 but was directed to the Selection API / Clipboard API repos — not sure what the interdependencies are, so I'm logging it here.

I think the Copy action needs some clarification on: 

> 3.1. Copy the selected contents, if any, to the clipboard. Implementations should create alternate text/html and text/plain clipboard formats when content in a web page is selected.

In regards to how the selection translates to html content. Case in point, browsers treat a Selection whose Range is set to either a Node (via `range.selectNode`) or a Node's content (via `range.selectNodeContent`) the same way:

* Firefox will copy the Node's inner HTML to the clipboard
* Chrome and Safari will copy the Node's outer HTML to the clipboard

I believe there needs to be a way for authors to specify if the Node as a whole should be included in the clipboard, or just its content.

**Comment by mbrodesser** [comment]
To achieve compatibility across browsers, it should additionally be specified how the selected contents are copied.

For instance, when selecting rows in a table, current browser behavior (Chrome, Firefox) when pasting them to the clipboard is to create a tab between the content of two adjacent cells.

Example: open the following in a browser, select all and copy-paste to a text editor:
```data:text/html,<table><tr><td>x</td><td>y</td><tr/> <tr><td>a</td> <td>b</td></tr> </table>```

However, if the selected elements have the display property `block`, Chrome and Firefox create line breaks between two adjacent cells.

Example: open the following in a browser, select all and copy-paste to a text editor:
```data:text/html,<table><tr><td><p>x</td><td><p>y</td><tr/> <tr> <td><p>a</td> <td><p>b</td> </tr> </table>```

Having such and potentially other related details specified would help achieving a reasonable and consistent behavior across browsers.

**Comment by EdgarChen** [comment]
>     * Firefox will copy the Node's inner HTML to the clipboard
> 
>     * Chrome and Safari will copy the Node's outer HTML to the clipboard

@danburzo, I guess you meant to say:
* Firefox will put the node's ***outer html*** to the clipboard's text/html mimetype
* Chrome and Safari will put the node's ***inner html*** to the clipboard

?

**Comment by danburzo** [comment]
Yes, that’s true. It seems I mixed the two when transcribing the issue from https://github.com/w3c/editing/issues/168

**Comment by EdgarChen** [comment]
We get a webcompat report that is related to this difference, see https://bugzilla.mozilla.org/show_bug.cgi?id=1876887#c14, though we think site needs change as well.
